connaisseur/config.py

```diff
@@ -40,7 +41,10 @@ def __init__(self):
                 "invalid format for Connaisseur configuration."
             )
 
-        self.notaries = [Notary(notary) for notary in config_content]
+        self.notaries = [
+            Notary(**{k: v for k, v in notary.items() if v is not None})
+            for notary in config_content
+        ]
```

Comment:
First, why are `None` values a problem?
Second, why take care of them here? I think `Notary.__init__` should handle that.

We can discuss off-band if you like.